stories/internal/DateFilter/DateFilter.tsx

```diff
@@ -0,0 +1,81 @@
+// (C) 2007-2019 GoodData Corporation
```

Comment:
why this file was created? There is already some DateFilter storybook in Helper components section, same as Attribute Filter. Why don't put it there?

Reply:
Git shows this as new file, but actually it is renamed file `DateFilterCore.tsx` to this new name `DateFilter.tsx`. So there was one story, I added second story named "localized".